### ISSUE TITLE
Failed install of sync dependency when XCode.app is named differently

### ISSUE THREAD

**Issue by cowgp** [realm/realm-cocoa#6827]
## Goals
Have multiple version of XCode installed at the same time and still be able to install realm dependency via cocoapods

## Expected Results
`pod install` succeeds

## Actual Results
`pod install` fails with
```
Installing Realm 5.4.7 (was 4.4.0 and source changed to `https://cdn.cocoapods.org/` from `trunk`)
[!] /bin/bash -c 
set -e
sh build.sh cocoapods-setup

Downloading dependency: sync 5.0.28 from https://static.realm.io/downloads/sync/realm-sync-cocoa-5.0.28.tar.xz
clang: error: no such file or directory: '12.0.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk'
clang: warning: no such sysroot directory: '/Applications/Xcode' [-Wmissing-sysroot]
```

## Steps for others to Reproduce
When preparing to update to new version of XCode I have my currently stable support version named XCode.app and then download and place in my applications folder addition version(s) of XCode but modify their app names to include version.  For example in my applications folder I currently have:

- XCode
- XCode 11.3.1
- XCode 12.0

If there is a space in the name of the currently in use version of XCode - then the error occurs.

Easy workaround is to not use spaces in app name.

## Version of Realm and Tooling

Realm framework version: 5.4.7

Realm Object Server version: ?

Xcode version: 12.0

iOS/OSX version: 10.15.6

Dependency manager + version: 1.9.3

**Comment by tgoyne** [comment]
Obviously we should fix this, but I definitely would recommend avoiding spaces in your Xcode app names.

**Comment by cowgp** [comment]
Yes, I'm in no way blocked by this  - just thought I'd point it out.
